src/main/java/edu/wpi/first/pathui/PathDisplayController.java

```diff
@@ -134,7 +130,7 @@ private void addPathToPane(Path newPath) {
     currentPath.set(newPath);
   }
 
-  private void addWaypointToPane(Waypoint current) {
+  protected void addWaypointToPane(Waypoint current) {
```

Comment:
Lets go with public for the functions you need to be public. With an accompanying javadoc comment.